### ISSUE TITLE
Telegram block access to the login by version numbers if it's a new session

### ISSUE THREAD

**Issue by arzezio7** [nikitasius/Telegraher#165]
The app does not work with Telegram servers anymore. It keeps telling to use new build.

**Comment by DI555** [comment]
no, it works. but login is blocked by telegram on all versions <= 9.3.3
have a patience! or use original client app for now.
So, wait for update!

**Comment by arzezio7** [comment]
Yes, sorry for incorrectly explaining about the situation. It's the login, not the app. I regretted uninstalling it in the first place just for troubleshooting.

**Comment by arzezio7** [comment]
I am currently using the original client app but some channels/groups are blocked and only with Telegraher I can get the full unrestricted experience.

**Comment by nikitasius** [comment]
Hi, there are 12.92.8 release is building, try it.
